[2025-02-18T04:31:21.014+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:31:21.021+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-04-25T00:00:00+00:00 [queued]>
[2025-02-18T04:31:21.023+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-04-25T00:00:00+00:00 [queued]>
[2025-02-18T04:31:21.024+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:31:21.028+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-04-25 00:00:00+00:00
[2025-02-18T04:31:21.031+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=452) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:31:21.031+0000] {standard_task_runner.py:72} INFO - Started process 453 to run task
[2025-02-18T04:31:21.031+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-04-25T00:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpgqkaxd2_']
[2025-02-18T04:31:21.032+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask hello_task
[2025-02-18T04:31:21.039+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:21.053+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-04-25T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:31:21.053+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:21.084+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-25T00:00:00+00:00'
[2025-02-18T04:31:21.085+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:31:21.090+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:31:21.091+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:31:21.095+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:31:21.095+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-04-25T00:00:00+00:00, execution_date=20240425T000000, start_date=20250218T043121, end_date=20250218T043121
[2025-02-18T04:31:21.130+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:31:21.137+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T04:31:21.138+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
